flytekit/exceptions/user.py

```diff
@@ -64,6 +64,15 @@ def __init__(self, path: str):
         super(FlyteDataNotFoundException, self).__init__(path, "File not found")
 
 
+class FlyteEntityNotFoundException(FlyteValueException):
+    def __init__(self, module_name: str, entity_name: str):
+        self._module_name = module_name
+        self._entity_name = entity_name
+
+    def __str__(self):
+        return f"Entity '{self._entity_name}' not found in module '{self._module_name}'"
```

Comment:
For a new flyte user, I do not think we explain what an "entity" is. I'm looking over the docs and it's hard to find a definition.

Reply:
how about "task" or "workflow"?

Reply:
"Task/Workflow '{self._entity_name}' not found in module"

Reply:
I'm +1 on `Task/Workflow`.